spec/support/capybara_drivers.rb

```diff
@@ -17,12 +17,24 @@ def assign_current_driver(driver)
     def register_driver(driver)
       Capybara.register_driver driver do |app|
         capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(chromeOptions: chrome_options)
-        Capybara::Selenium::Driver.new(app, browser: :chrome, desired_capabilities: capabilities)
+        if ENV['SELENIUM_URL']
+          Capybara.server_port = '3002'
+          Capybara.server_host = '0.0.0.0'
+          Capybara.app_host = "http://#{ENV['APP_HOST']}:#{Capybara.server_port}"
+          Capybara::Selenium::Driver.new(app,
+                                         browser: :remote,
+                                         desired_capabilities: capabilities,
+                                         url: ENV['SELENIUM_URL'].to_s)
+        else
+          Capybara::Selenium::Driver.new(app,
+                                         browser: :chrome,
+                                         desired_capabilities: capabilities)
+        end
       end
     end
 
     def chrome_options
-      { args: ['no-sandbox', 'disable-gpu', 'window-size=1024,768', 'single-process'] }
+      { args: ['headless', 'no-sandbox', 'disable-gpu', 'window-size=1024,768', 'single-process'] }
```

Comment:
@whereismyjetpack does 'headless' actually work? If you take it out, does it _not_ run headless?

Reply:
I seem to remember needing to put this in. My memory is a bit hazy right now though :)

Reply:
https://drone-test.dsrd.libraries.psu.edu/psu-stewardship/scholarsphere-4/392/1/9 taking headless out does, indeed fail it.